content/applications/marketing/email_marketing.rst

```diff
@@ -1,440 +1,792 @@
+- :ref:`Kanban <email_marketing/kanban-view>`
+- :ref:`Calendar <email_marketing/calendar-view>`
+- :ref:`Graph <email_marketing/graph-view>`
 
-The four kanban columns each represent the various stages of emails that have been built or sent
-within the *Email Marketing* application.
+.. _email_marketing/list-view:
+
+List view
+---------
 
-The stages are: :guilabel:`Draft`, :guilabel:`In Queue`, :guilabel:`Sending`, :guilabel:`Sent`.
+The list view, represented by the :guilabel:`☰ (horizontal lines)` icon in the upper-right corner,
+is the default view of the :guilabel:`Mailings` dashboard in the *Email Marketing* app.
 
-- :guilabel:`Draft`: means the email is still being written/created.
-- :guilabel:`In Queue`: means the email is scheduled, and sent at a later date.
-- :guilabel:`Sending`: means the email is currently being sent to its recipients.
-- :guilabel:`Sent`: means the email has already been sent to its recipients.
+While in list view, there are columns dedicated to different aspects of information related to the
+listed emails. Those columns are as follows:
 
-In each stage, there are drag-and-drop cards representing the email campaigns that have been
-created, and the stage they are in represents the current status of that mailing.
+- :guilabel:`Date`: the date the email was sent.
+- :guilabel:`Subject`: the subject of the email.
+- :guilabel:`Responsible`: the user who created the email, or the user who has been assigned to the
+   email.
+- :guilabel:`Sent`: how many times the email has been sent.
+- :guilabel:`Delivered (%)`: percentage of sent emails that have been successfully delivered.
+- :guilabel:`Opened (%)`: percentage of sent emails that have been opened by the recipients.
+- :guilabel:`Clicked (%)`: percentage of sent emails that have been clicked by the recipients.
+- :guilabel:`Replied (%)`: percentage of sent emails that have been replied to by the recipients.
+- :guilabel:`Status`: the status of the email (:guilabel:`Draft`, :guilabel:`In Queue`, or
+  :guilabel:`Sent`).
 
-Each mailing block on the :guilabel:`Email Marketing` dashboard provides key information related to
-that specific email.
+To add or remove columns, click the :guilabel:`Additional Options (two horizontal lines with dots)`
+icon, located to the far-right of the column titles in list view. Doing so reveals a drop-down menu
+of additional column options.
 
-When the cursor hovers over the upper-right corner of an email campaign card, three vertical dots
-(:guilabel:`⋮`) appear. When clicked, a mini drop-down menu reveals the option to color-code the
-email :guilabel:`Delete` the email altogether, or :guilabel:`Archive` the message for potential
-future use.
+.. _email_marketing/kanban-view:
+
+Kanban view
+-----------
+
+The Kanban view, represented by the :guilabel:`(inverted bar graph)` icon, can be accessed in the
+upper-right corner of the :guilabel:`Mailings` dashboard in the *Email Marketing* application.
```

Comment:
```suggestion
upper-right corner of the :guilabel:`Mailings` dashboard in the :guilabel:`Email Marketing` app.
```

Reply:
So, this change is kinda "news" to me...So, even if it just appears in the header as the app title, we should always use a GUI? I wasn't aware of that. I'm implementing all these changes, but just wanna know if this is going to be the standard moving forward.

Reply:
it will also help me in my Final Reviews to know this, as well. Thanks

Reply:
Sure! I am happy to discuss this because I want to be on the same page as well! 

The criteria that I look for in order to use a :guilabel: is to ask: is the text on the screen during this active instruction, if so, use a :guilabel: for the text, otherwise italics can be used.

Our [guidelines](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#gui-element) state:
> Use the guilabel markup to identify any text of the interactive user interface

So while we do italics on app names (i.e., _Email Marketing_), in this instance on line 86, the text "Email Marketing" is present on the UI in this instruction, so thats why I am suggesting to use a :guilabel:.

Let me know your thoughts, and tagging @StraubCreative here as well so we can discuss. Thanks!